specification/databricks/resource-manager/Microsoft.Databricks/preview/2021-04-01-preview/databricks.json

```diff
@@ -652,6 +652,52 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Databricks/workspaces/{workspaceName}/OutboundNetworkDependenciesEndpoints": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Gets a list of egress endpoints (network endpoints of all outbound dependencies) in the specified Workspace.",
+        "description": "Gets a list of egress endpoints (network endpoints of all outbound dependencies) in the specified Workspace. The operation returns properties of each egress endpoint.",
```

Comment:
Should say currently it is available in public. But for MC and FF this endpoint return empty quotes

Reply:
Can we get better description. Here is description
"Gets the list of endpoints that VNET Injected Workspace's Cluster call Azure Databricks Control Plane. You must configure outbound access to these endpoints. For more information, see https://docs.microsoft.com/en-us/azure/databricks/administration-guide/cloud-configurations/azure/udr"

Reply:
Changed Description